app/serializers/articles/show_serializer.rb

```diff
@@ -1,3 +1,15 @@
 class Articles::ShowSerializer < ActiveModel::Serializer
+  include ActionView::Helpers::TextHelper
+
   attributes :id, :title, :body
-end
\ No newline at end of file
+
+  def body
+    if current_user != nil && current_user.role != nil
```

Comment:
should a user with the role of `:user` see the whole article?

Reply:
We have set default role to nil in previous PR. :user is no more.

Reply:
ok cool